pywatts/conditions/cd_condition.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import xarray as xr
+from river.drift import ADWIN
+
+from pywatts.core.base_condition import BaseCondition
+
+
+class RiverDriftDetectionCondition(BaseCondition):
+    """
+    A Drift Detection Condition that wraps the detection algorithms from the River library, based on the RMSE metric.
+
+    :param drift_detection: The Drift Detection Algorithm from the River library. The default algorithm is ADWIN.
+    """
+
+    def __init__(self, name="CDCondition", drift_detection=ADWIN()):
+        super().__init__(name=name)
+        self.drift_occured = False
+        self.drift_detection = drift_detection
+        self.counter = 0
+
+    def evaluate(self, y: xr.DataArray, y_hat: xr.DataArray):
+        """
+        Returns True if the specified drift detection algorithm detects a drift.
+        :param y: GT Time series
+        :type y: xr.DataArray
+        :param y_hat: Forecast Time Series
```

Comment:
I suggest to unify the upper lower case (Time series, Time Series) and to write out the acronym GT